doc/surveys.md

```diff
@@ -0,0 +1,57 @@
+# Running test surveys on GOV.UK
+
+Across GOV.UK, a "survey" will pop up for one in 50 visitors, asking them what they thought of GOV.UK. This is known as the "user satisfaction survey".
+
+Sometimes, a "small survey" or "test survey" needs to be run across GOV.UK for a short time to evaluate content changes or taxonomy changes.
+
+This can easily be done by adding an entry to `GOVUK.userSurveys.smallSurveys` in `app/assets/javascripts/surveys.js`.
+
+## Example
+
+```javascript
+{
+  url: 'https://www.surveymonkey.com/s/2AAAAAA/',
+  identifier: 'education_only_survey',
+  frequency: 50,
+  template: TEMPLATE,
+  activeWhen: function() {
+    return $('meta[property="govuk:section"]').attr('content') === 'education and learning';
```

Comment:
I'll add a single example function, to show how they can be defined inline. Good shout.